### PR TITLE
Update to RuboCop 1.1

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by phiggins** [comment]
I like it. Using `Chef/` as the namespace makes a lot of sense.

**Review by phiggins — APPROVED** [review]
I'm surprised enabling so many cops didn't result in more violations. 👍

**Comment by tas50** [comment]
Nothing new actually gets enabled. We take the base config, then apply a config that disables them all and then selectively enable them.

**Comment by tas50** [comment]
@jaymzh With this chance you'll get a new `--only Chef` capability to resolve https://github.com/chef/cookstyle/issues/734

File: tasks/cops_documentation.rake [hunk 1]
```diff
@@ -204,41 +200,31 @@ begin
...
-        .sort
-        .map { |department| table_of_content_for_department(cops, department) }
-        .reject(&:nil?)
-        .join("\n")
-    end
-
     def main
-      cops   = RuboCop::Cop::Cop.registry
+      all_cops = RuboCop::Cop::Cop.registry
+      chef_departments = all_cops.departments.reject { |d| !d.start_with?('Chef') }.sort
```
Comment (phiggins) [inline tasks/cops_documentation.rake:219]:
```suggestion
      chef_departments = all_cops.departments.select { |d| d.start_with?('Chef') }.sort
```

**Comment by tas50** [comment]
This was done a while back

**Pull request merged**